beacon_node/network/src/sync/tests/range.rs

```diff
@@ -116,7 +121,7 @@ impl TestRig {
                     NetworkMessage::SendRequest {
                         peer_id,
                         request: RequestType::BlobsByRange(_),
-                        request_id: AppRequestId::Sync(SyncRequestId::RangeBlockAndBlobs { id }),
+                        request_id: AppRequestId::Sync(SyncRequestId::BlobsByRange(id)),
```

Comment:
Just a heads up that I've added data column handling for these tests - if this one gets approved first I'll address it in my PR :)